src/contrib/cluster/Akka.Cluster.Metrics/Helpers/DynamicAccess.cs

```diff
@@ -0,0 +1,25 @@
+// //-----------------------------------------------------------------------
+// // <copyright file="DynamicAccess.cs" company="Akka.NET Project">
+// //     Copyright (C) 2009-2019 Lightbend Inc. <http://www.lightbend.com>
+// //     Copyright (C) 2013-2019 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// // </copyright>
+// //-----------------------------------------------------------------------
+
+using System;
+
+namespace Akka.Cluster.Metrics.Helpers
+{
+    /// <summary>
+    /// DynamicAccess
+    /// </summary>
+    internal static class DynamicAccess
```

Comment:
Good idea. Also I will add `Try` class to `Akka.Utils`, since it is used by `DynamicAccess` and also may be used somewhere else.

Reply:
Might also be worth getting rid of `DynamicAccess` altogether and just creating some `Activator` extension methods - more idiomatic that way. What do you think?

Reply:
Mmm what do you mean by `Activator` extension methods? Obviously is can not add custom static method like `Activator.CreateInstanceFor<TSomeInterface>(string typeName)`. Either use `(new Activator()).CreateInstanceFor`, or create another static class like `ActivatorHelper` - but I named it `DynamicAccess` to just simplify porting scala code in the future (sometimes one needs to check if some akka's class is already implemented, and search by name helps in such cases).

We can also add extension method for string, like `typeName.CreateInstanceAs<TInterface>(params object[] args)`, but... It would be hard to find for other developers, and not obvious.

So since  `DynamicAccess.CreateInstanceFor` wraps getting type by name, then instantiating with `Activator`, and handling exceptions (it returns `Try<TResult>` instance which may contain error info), it might be useful when porting things in future. But I can rename it to something more intuitive - do you have any suggestions? I agree that nobody would look for `DynamicAccess` class if only he is not porting `dynamicAccess` class from scala at the moment.

Reply:
@IgorFedchenko let's make `DynamicAccess` `internal` and then keep it as-is - that way if we decided to clean up the API to make it more idiomatic with .NET in the future we can do that without worrying about user-facing issues.

Reply:
To keep it in `Akka.Utils` to make reusable in different modules marked it with `[InternalApi]` and added `INTERNAL USAGE` text into class summary.